presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergDistributedQueries.java

```diff
@@ -223,4 +238,23 @@ public void testRenameViewIfNotExists()
                 format("line 1:1: View '%s.%s.test_rename_view_not_exist' does not exist", catalog, schema));
         assertQuerySucceeds("ALTER VIEW IF EXISTS test_rename_view_not_exist RENAME TO test_renamed_view_not_exist");
     }
+
```

Comment:
This class is for more generic e2e test queries. There has been a separate file to test the changelog function. It may make sense to rename TestApplyChangeLongFunction to TestIcebergFunctions, then add bucket function tests to it.